sdk/src/feature_set.rs

```diff
@@ -49,6 +49,15 @@ pub mod full_inflation {
         }
     }
 
+    pub mod bunghi {
+        pub mod vote {
+            solana_sdk::declare_id!("E9hFUVEz29H8XMXk7ygk7ZpCuEuZQ8DJvJKJSTGu1RM6");
+        }
+        pub mod enable {
+            solana_sdk::declare_id!("5S9JDUb4vKY1CUxLf5oc96ZxjGrephj1jcPeTi62sYmP");
+        }
+    }
+    
```

Comment:
hmm, it's still unhappy.  This is the change it wants I think:
```suggestion

```

Reply:
@bunghi - I'll make the change here, don't push a new commit.  Let's see if this works...